test/indexer/test_tdr.py

```diff
@@ -68,6 +78,10 @@ class TestTDRPlugin(CannedBundleTestCase):
     def tinyquery(self) -> tinyquery.TinyQuery:
         return tinyquery.TinyQuery()
 
+    @cache
+    def plugin_for_source_name(self, source_name) -> tdr.Plugin:
+        return TestPlugin(sources={source_name}, tinyquery=self.tinyquery)
```

Comment:
Might be able to instantiate one and reuse it, which i guess in a way this does with `@cache`...
🤔

Reply:
Currently we only use one source for the tests. I don't have any concrete reason to expect that to change. But it might and if it does this is simple way to facilitate that.